x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/translations.ts

```diff
@@ -126,6 +126,20 @@ export const THRESHOLD_RESULTS_AGGREGATED_BY = i18n.translate(
   }
 );
 
+export const THRESHOLD_CARDINALITY = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleDescription.thresholdResultsCardinalityDescription',
+  {
+    defaultMessage: 'when unique value count of',
```

Comment:
```suggestion
    defaultMessage: 'when unique values count of',
```

Reply:
I think we should use here translation interpolation, to make this easier to translate without losing context.

```ts
  i18n.translate(
    'xpack.securitySolution.detectionEngine.ruleDescription.thresholdResultsCardinalityDescription',
    {
      defaultMessage:
        '{thresholdFieldsGroupedBy} when unique values count of {cardinalityField} >= {cardinalityValue}',
      values: {
        thresholdFieldsGroupedBy,
        cardinalityField,
        cardinalityValue,
      },
    }
  );

```

Reply:
It can be a function, refer to implementation in [similar cases](https://github.com/elastic/kibana/blob/8.16/x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/prebuilt_rules/translations.ts), or it can be `FormattedMessage` component, as in https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/modals/ml_job_upgrade_modal/translations.tsx

A bit more on interpolation and FormattedMessage component: https://github.com/elastic/kibana/blob/8.16/packages/kbn-i18n/GUIDELINE.md#in-reactjs

Reply:
Ok, I will improve this.
The text however was suggested by @approksiu. I will double check with her if this is OK.